### ISSUE TITLE
User interfaces do not support longer texts, violating the accessibility law.

### ISSUE THREAD

**Issue by GabrieleSuzart** [flutter-stripe/flutter_stripe#299]
## **Describe the bug**
The text size of the add credit card screen is fixed, and font accessibility does not support.

## **To Reproduce**
1. Open device settings
2. Enter Accessibility
3. Enter Screen and Text Size
4. Activate the larger texts option and slide the slider to maximum size
5. Open the screens and verify that the font size does not match the setting selected by the user

## **Expected behavior**
- I expect the fonts to get bigger when, as per the device's size settings. (Fields must be vertical for larger fonts.)
- I expect the icon to grow according to the font size.

## **Smartphone / tablet**
 - Device: iPhone Xr
 - OS: iOS 14.7.1
 - Package version: 1.2.0
 - Flutter version 2.2.3

## **Additional context**
This accessibility criterion is AA compliance level and is therefore required by law. In case of audit, the company must pay a fine and correct the problem.
Check:
https://www.w3.org/WAI/WCAG21/Understanding/resize-text
https://www.w3.org/WAI/WCAG21/Understanding/reflow

Flutter takes the percentage of fonts in: `MediaQuery.of(context).textScaleFactor` 
Examples: 1 = 100% and 3.5 = 350%

## Print
![simulator_screenshot_40CC3E8D-D1C5-49A9-BAD5-8C62F198E224](https://user-images.githubusercontent.com/30198542/130598348-02282e66-259c-490f-b901-0e23db76bbad.png)

**Comment by remonh87** [comment]
We use native stripe sdk field so I need to check if we can for some reason tweak this.

As alternative you can checkout the paymentSheet which has a bit more build in utilities (of course it needs to fit your demands).

**Comment by deep8292** [comment]
The same is happening with me and it does not show a placeholder on Android, @remonh87 may we know an ETA of the new update with such fixes?
